src/MobileBookingHeader/ReturnTrip.js

```diff
@@ -0,0 +1,48 @@
+// @flow
+
+import * as React from 'react';
+import { createFragmentContainer, graphql } from 'react-relay';
+
+import ReturnTrip_booking from './__generated__/ReturnTrip_booking.graphql';
+import { TripDescriptionStyle } from './commonStyles';
+
+type Props = {|
+  booking: ReturnTrip_booking,
+|};
+
+const ReturnTrip = ({ booking }: Props) => {
+  const origion = booking.outbound.departure.airport.city.name;
```

Comment:
`idx` maybe?

Reply:
typo "origion"

Reply:
How is it possible that Flow is happy with this... 😱